instrumentation/afl-compiler-rt.o.c

```diff
@@ -350,17 +369,18 @@ static void __afl_map_shm(void) {
 
     }
 
+    close(shm_fd);
+
```

Comment:
suggest also moving:
`shm_fd = -1;`
from line 376

Reply:
moved it, however should rather remove it, it is not used afterwards (if, it would be a bug)